### PR TITLE
[docs] Small changes to standardize acessibility section for Data Grid and Tree View

### PR DESCRIPTION
**Pull request by arthurbalduini**
Some nitpicks I found while working on the acessibility documentation for pickers.

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui/mui-x/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

File: docs/data/data-grid/accessibility/accessibility.md [hunk 1]
```diff
@@ -106,17 +106,23 @@ renderCell: (params) => (
...
+- <kbd class="key">End</kbd> with <kbd class="key">fn</kbd>+<kbd class="key">Arrow Right</kbd>
+- <kbd class="key">Page Up</kbd> with <kbd class="key">fn</kbd>+<kbd class="key">Arrow Up</kbd>
+- <kbd class="key">Page Down</kbd> with <kbd class="key">fn</kbd>+<kbd class="key">Arrow Down</kbd>
+:::
 
 |                                                               Keys | Description                                                 |
 | -----------------------------------------------------------------: | :---------------------------------------------------------- |
 |                                  <kbd class="key">Arrow Left</kbd> | Navigate between cell elements                              |
-|                                <kbd class="key">Arrow Bottom</kbd> | Navigate between cell elements                              |
+|                                  <kbd class="key">Arrow Down</kbd> | Navigate between cell elements                              |
```
Comment (flaviendelangle) [inline docs/data/data-grid/accessibility/accessibility.md:125]: The famouse "Arrow Bottom" :laughing:

**Review by flaviendelangle — APPROVED** [review]
Nice improvement :ok_hand:

**Comment by flaviendelangle** [comment]
For the failing CI, rebasing on `master` should fix it

File: docs/data/tree-view/accessibility/accessibility.md [hunk 2]
```diff
@@ -31,8 +31,10 @@ The [WAI-ARIA Authoring Practices](https://www.w3.org/WAI/ARIA/apg/patterns/tree
 :::info
 The following key assignments apply to Windows and Linux users.
 
-On macOS replace <kbd class="key">Ctrl</kbd> with <kbd class="key">⌘ Command</kbd>.
-
+On macOS replace:
```
Comment (LukasTy) [inline docs/data/tree-view/accessibility/accessibility.md:34]: I'm not sure how correct we are here. 🤔 
Those keys definitely exist on MacOS and their keyboards:
https://store.storeimages.cdn-apple.com/4982/as-images.apple.com/is/MQ052?wid=2000&hei=2000&fmt=jpeg&qlt=95&.v=1495129815011
It's only MacBooks with their limited amount of keys or any other keyboard with less keys, which remove less often used keys. 🤔 
So, technically, with this we are educating users about MacOS shortcuts for keys that they MIGHT not have on their keyboard.
I'm not saying that we can't do it. Just questioning if maybe we could put a bit less emphasis on it and instead only link to a MacOS keyboard shortcut page? 🤔

**Review by LukasTy — COMMENTED** [review]
Please sync with latest upstream/master to fix CI. 😉

**Review by noraleonte — APPROVED** [review]
Very nice catch 🎉

File: docs/data/tree-view/accessibility/accessibility.md (hunk 2 above)
Comment (LukasTy) [reply docs/data/tree-view/accessibility/accessibility.md:34]: It makes sense, we can definitely put those additions as improvements that would have less emphasis than actually different buttons. 👍

File: docs/data/date-pickers/accessibility/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+productId: x-date-pickers
+title: Date and Time Pickers - Date format and localization
```
Comment (LukasTy) [inline docs/data/date-pickers/accessibility/accessibility.md:3]: Did you intend to add this page on this PR? 🤔 
If so, then this title should better reflect the page content and maybe we'd need to adjust the PR title. 😉

**Review by LukasTy — COMMENTED** [review]
Consider merging latest `upstream/master` into your branch to avoid unwanted diffs.
Running `prettier` should fix the CI issues. 😉

File: docs/data/date-pickers/accessibility/accessibility.md (hunk 3 above)
Comment (arthurbalduini) [reply docs/data/date-pickers/accessibility/accessibility.md:3]: Looks like I did something wrong 🙈

**Pull request closed without merging**